### PR TITLE
[Backport to 2.8.x] Fix for #3872 - Theme refactoring

### PR DESCRIPTION
**Pull request by afabiani**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -81,3 +81,6 @@ celerybeat-schedule.*
 geonode\.tests\.bdd\.e2e\.test_login/
 
 *.log
+
+/celerybeat.pid
+/celerybeat-schedule
```
Comment (olivierdalang) [inline .gitignore]: these changes are not related are they ?

Comment (afabiani) [reply .gitignore]: Yep, sorry. Will remove it.

**Comment by olivierdalang** [comment]
@mikefedak Thanks for the report, but how did this happen to you ? the commit also deletes `geonode/admin.py`

**Comment by mikefedak** [comment]
It does delete admin.py but not admin.pyc
